src/routes/index.tsx

```diff
@@ -1,30 +1,13 @@
 import React from 'react'
-import { Route, Link } from 'react-router-dom'
+import { Route } from 'react-router-dom'
+
+// Pages
```

Comment:
There are some comments categorized as "bookmarks" but in this case, I wouldn't think it's necessary because we can already infer the context of the modules by reading it's paths